javascript/javascript3/week1/giphy.js

```diff
@@ -0,0 +1,48 @@
+//35jNtGWDxwmuaOlAotfLRXM2RHo9Vwh2
+//https://api.giphy.com/v1/gifs/search?api_key=35jNtGWDxwmuaOlAotfLRXM2RHo9Vwh2&q=smile&limit=5&rating=g
+
+const displayImages = document.getElementById("displayImages");
+const numberOfGiphy = document.getElementById("numberOfGiphy");
+const btnSearch = document.getElementById("btnSearch");
+
+function getGifImages(limit) {
+  const searchText = document.getElementById("searchGiphy");
+  if (!/^\s*$/.test(searchText.value)) {
+    fetch(
+      "https://api.giphy.com/v1/gifs/search?api_key=35jNtGWDxwmuaOlAotfLRXM2RHo9Vwh2&q=" +
```

Comment:
Save this to a variable that you append to. Also, keep your API key in a secret file that you don't push. Another cool thing you can do is creating a dedicated function to formulate your query to the API.